.github/workflows/wheels.yml

```diff
@@ -0,0 +1,152 @@
+name: Build and test wheels
+
+on:
+  push:
+    branches:
+      - master
+      - test
+  pull_request:
+  create:
+
+jobs:
+  windows:
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        python: [3.6, 3.7, 3.8]
+        wordsize: [32, 64]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Install deps
+        env:
+          PYTHON: "py -${{ matrix.python }}-${{ matrix.wordsize }}"
+        shell: bash
+        run: |
+          set -ex
+          ${PYTHON} -m pip install --upgrade pip
+          ${PYTHON} -m pip install setuptools wheel
+          ${PYTHON} -m pip install -r python/requirements/CI/wheel-build/requirements.txt
+      - name: Build C extension
+        env:
+          PYTHON: "py -${{ matrix.python }}-${{ matrix.wordsize }}"
+        shell: bash
+        run: |
+          set -ex
+          # Fix symlinks on windows
+          git config core.symlinks true
+          git reset --hard
+          cd python
+          ${PYTHON} -VV
+          ${PYTHON} setup.py build_ext --inplace
+          ${PYTHON} -c "import kastore"
```

Comment:
If you want to check if the binary module built, better to ``import _kastore``. There's code in the high-level module to conditionally import the low-level module (as it's not actually strictly necessary to do stuff).

Reply:
Fixed in https://github.com/tskit-dev/kastore/pull/118/commits/5a26733548f9b85c0ad7c731efdf72aacc57a92c